pkg/system/phase2_creating.go

```diff
@@ -52,6 +52,22 @@ func (r *Reconciler) ReconcilePhaseCreating() error {
 		return err
 	}
 
+	if err := r.ReconcileObject(r.RouteMgmt, nil); err != nil {
+		if meta.IsNoMatchError(err) || runtime.IsNotRegisteredError(err) {
```

Comment:
@nb-ohad After you rebase from master, see that I created a function to `ReconcileObjectOptional()` so no need to copy these conditions over and over.